File: Tasks/Python Work/temperature_conversion.py
for counter in range (5):

	temperature_in_celsius = float(input("Enter a temperature in celsius: "))

	temperature_in_fahrenheit = (temperature_in_celsius *(9/5)) + 32

	if (temperature_in_celsius <= -273):
		print("Impossible!")

	else:
		print("Celsius to Fahrenheit = (",temperature_in_celsius,"x (9/5)) + 32 =",temperature_in_fahrenheit)

File: Tasks/Java Work/100 Tasks/TemperatureConversion.java
import java.util.Scanner;
	public class TemperatureConversion{
		public static void main(String[] args){
		Scanner input = new Scanner(System.in);

System.out.println("Enter a temperature in Celcius:");
double temperature = input.nextDouble();

double fahrenheit = 0;

fahrenheit = (temperature * (9/5.0)) + 32;

System.out.println("The temperature in Fahrenheit = " + fahrenheit);
}
}
